updating/updating-restricted-network-cluster/mirroring-image-repository.adoc

```diff
@@ -0,0 +1,171 @@
+:_content-type: ASSEMBLY
+[id="mirroring-ocp-image-repository"]
+= Mirroring the OpenShift Container Platform image repository
+include::_attributes/common-attributes.adoc[]
+:context: mirroring-ocp-image-repository
+
+toc::[]
+
+You must mirror container images onto a mirror registry before you can update a cluster in a disconnected environment. You can also use this procedure in connected environments to ensure your clusters only use container images that have satisfied your organizational controls on external content.
+
+[NOTE]
+====
+This registry must be running at all times as long as the cluster is running.
+====
+
+There are two supported methods for mirroring images onto a mirror registry:
+
+* Using the oc-mirror OpenShift CLI (`oc`) plugin
+
+* Using the `oc adm release mirror` command
+
+Compared to using `oc adm release mirror`, the oc-mirror plugin has the following advantages:
+
+* It is simpler to use.
+
+* It can mirror content other than container images.
+
+* After mirroring images for the first time, it is easier to update images in the registry.
```

Comment:
We should mention that , oc-mirror provides an automated way of mirroring and performs steps required for the disconnected environment.

Reply:
Is there a way to word this in a way that gives users an idea of what's being automated? Which parts of the old oc adm release mirror process are being completed for users when they use oc-mirror?

Reply:
Sure, we can mention `oc-mirror provides an automated way to mirror release payload from quay.io and also builds latest graph-data image for the OSUS running in disconnected environment.`